oci/config.py

```diff
@@ -1,46 +1,46 @@
 from collections import namedtuple
-import ConfigParser
 import os
+from six.moves import configparser
 
-gerrit_default_host = 'gerrit.ovirt.org'
-jenkins_default_host = 'jenkins.ovirt.org'
+DEFAULT_CONFIG = {
+    ('jenkins', 'host', 'jenkins.ovirt.org'),
+    ('gerrit', 'host', 'gerrit.ovirt.org')}
```

Comment:
This works because we have one default per section, but will break once we add the next default.

But I think it is good enough for now.